arch/arm/configs/tsimx6ul_defconfig

```diff
@@ -676,6 +676,9 @@ CONFIG_SPI_GPIO=y
 CONFIG_SPI_IMX=y
 CONFIG_SPI_OCORES=m
 CONFIG_SPI_SPIDEV=y
+CONFIG_PPS=m
+CONFIG_PPS_CLIENT_LDISC=m
+CONFIG_PPS_CLIENT_GPIO=m
```

Comment:
@markfeathers were these options added to the CI checks?

Reply:
Yes, we make sure pps0 is created on both kernel configs.